decidim-blogs/app/forms/decidim/blogs/admin/post_form.rb

```diff
@@ -10,13 +10,15 @@ class PostForm < Decidim::Form
         translatable_attribute :title, String
         translatable_attribute :body, String
 
-        attribute :user_group_id, Integer
+        attribute :user_group_id, String
 
         validates :title, translatable_presence: true
         validates :body, translatable_presence: true
 
         def map_model(post)
           self.user_group_id = post.author.id if post.author.is_a?(Decidim::UserGroup)
+          self.user_group_id = "current_organization" if post.author.is_a?(Decidim::Organization)
```

Comment:
This looks like a place to use a `case` statement, something like:
```ruby
self.user_group_id = case post.author
when Decidim::UserGroup
  post.author.id
...
```

Reply:
Pending: Thank you for the suggestion. We still need to incorporate the case statement.

Reply:
Done